File: VoiceRecorder/ShuttingDownMessage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace VoiceRecorder
{
    class ShuttingDownMessage
    {
        public string CurrentViewName { get; private set; }
        public ShuttingDownMessage(string currentViewName) 
        {
            this.CurrentViewName = currentViewName;
        }
    }
}

File: ~VB Port/VoiceRecorder/ShuttingDownMessage.vb
Imports System.Text

Namespace VoiceRecorder
    Friend Class ShuttingDownMessage
        Private _currentViewName As String
        Public Property CurrentViewName As String
            Get
                Return _currentViewName
            End Get
            Private Set(ByVal value As String)
                _currentViewName = value
            End Set
        End Property

        Public Sub New(ByVal currentViewName As String)
            Me.CurrentViewName = currentViewName
        End Sub
    End Class
End Namespace
